pkg/controller/dgraph/dgraph.go

```diff
@@ -162,3 +180,12 @@ func unmarshalDgraphResponse(resp *api.Response, id string) string {
 
 	return r.IDs[0].UID
 }
+
+func startCronGoRoutines() {
```

Comment:
This is especially for the deletion purpose, so let's rename it to something like `startDeleteResourceCronJob`. 
And I am not sure if we should place the Cron job in the Dgraph package.

Reply:
For now delete is the only function we need to execute daily. May be later we need to sync up complete inventory into dgraph. Keeping all dgraph related code within dgraph directory will help us if we want to use a flag to enable or disable usage of DB.